### PR TITLE
[WIP][ADD] account_payment_order_sequence_payment_mode

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**

### CODE REVIEW

File: account_payment_order_sequence_payment_mode/models/account_payment_mode.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountPaymentMode(models.Model):
+
+    _inherit = "account.payment.mode"
+
+    sequence_payment_order = fields.Many2one(
```
Comment (acsonefho) [inline account_payment_order_sequence_payment_mode/models/account_payment_mode.py]: Should be `sequence_payment_order_id` (+ related changes)

File: account_payment_order_sequence_payment_mode/models/account_payment_order.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    _inherit = "account.payment.order"
+
+    @api.model
+    def create(self, vals):
+        if vals.get("payment_mode_id"):
+
+            mode_id = vals.get("payment_mode_id")
+            seq = (
+                self.env["account.payment.mode"].browse(mode_id).sequence_payment_order
```
Comment (acsonefho) [inline account_payment_order_sequence_payment_mode/models/account_payment_order.py]: Missing readme for this module.

**Review by acsonefho — CHANGES_REQUESTED** [review]

File: account_payment_order_sequence_payment_mode/views/account_payment_mode.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_payment_mode_form" model="ir.ui.view">
+        <field name="name">account_payment_order.account.payment.mode.form</field>
+        <field name="model">account.payment.mode</field>
+        <field name="inherit_id" ref="account_payment_mode.account_payment_mode_form" />
+        <field name="arch" type="xml">
+            <field name="payment_type" position="after">
+                <field
+                    name="sequence_payment_order"
```
Comment (acsonefho) [inline account_payment_order_sequence_payment_mode/views/account_payment_mode.xml]: You forgot to rename the field in the view

**Review by acsonefho — CHANGES_REQUESTED** [review]

**Review by acsonefho — APPROVED** [review]

**Review by adrienpeiffer — APPROVED** [review]

**Pull request merged**